packages/core/framework/src/http/middlewares/error-handler.ts

```diff
@@ -20,14 +20,20 @@ export function errorHandler() {
     res: Response,
     _: NextFunction
   ) {
-    const logger = req.scope.resolve(ContainerRegistrationKeys.LOGGER)
+    const logger = req.scope
+      ? req.scope.resolve(ContainerRegistrationKeys.LOGGER)
+      : console
 
-    err = formatException(err)
+    if (!req.scope) {
+      logger.error(
+        "req.scope is missing unexpectedly. It should be defined in all the cases"
```

Comment:
suggestion: if this is an error that the user might face, I think the error should be more helpful to the user, like what should they do if they see this?

Reply:
This is not really for the user, but for us to dig further into the error and see why `req.scope` is undefined.

For example, currently it was `undefined` because we were not able to connect to DynamoDB because the user had defined some env vars in their cloud dashboard, that they shouldn't define.